lib/timers.js

```diff
@@ -87,13 +87,20 @@ function listOnTimeout() {
 
   var first;
   while (first = L.peek(list)) {
-    // If the previous iteration caused a timer to be added,
-    // update the value of "now" so that timing computations are
-    // done correctly. See test/simple/test-timers-blocking-callback.js
-    // for more information.
+    // Whenever a timer creates a new timer in the current list being processed
```

Comment:
You got it.  Should I also update the commit message as well?  The code comment would make an excellent description in the Git commit message if you ask me.